src/plugins/wizard/public/application/components/data_tab/field_selector_field.scss

```diff
@@ -0,0 +1,12 @@
+.wizFieldSelectorField {
+  @include euiBottomShadowSmall;
+  padding: $euiSizeXS;
+  background-color: $euiColorEmptyShade;
+  border: $euiBorderThin;
+  margin-top: $euiSizeS;
+
+  & > button {
+    align-items: center;
+    gap: 4px;
```

Comment:
Can this come from the theme too?

Reply:
The style for this is yet to be updated. @btzeng is still working on the final style for this section